bedboss/bedmaker/bedmaker.py

```diff
@@ -60,65 +61,78 @@ def __init__(
         **kwargs,
     ):
         """
-        A pipeline to convert bigwig, bedGraph, bed, bigBed or wig files into bed and bigBed format
+        Pypiper pipeline to convert supported file formats into
+        BED format and bigBed format. Currently supported formats*:
+            - bedGraph
+            - bigBed
+            - bigWig
+            - wig
         :param input_file: path to the input file
-        :param input_type: a [bigwig|bedgraph|bed|bigbed|wig] file that will be converted into BED format
+        :param input_type: a [bigwig|bedgraph|bed|bigbed|wig] file that will be
+                           converted into BED format
         :param output_bed: path to the output BED files
         :param output_bigbed: path to the output bigBed files
-        :param sample_name: name of the sample used to systematically build the output name
+        :param sample_name: name of the sample used to systematically build the
+                            output name
         :param genome: reference genome
         :param rfg_config: file path to the genome config file
-        :param chrom_sizes: a full path to the chrom.sizes required for the bedtobigbed conversion
-        :param narrowpeak: whether the regions are narrow (transcription factor implies narrow, histone mark implies broad peaks)
+        :param chrom_sizes: a full path to the chrom.sizes required for the
+                            bedtobigbed conversion
+        :param narrowpeak: whether the regions are narrow (transcription factor
+                           implies narrow, histone mark implies broad peaks)
         :param sntandard_chrom: Standardize chromosome names. Default: False
         :param check_qc: run quality control during badmaking
         :return: noReturn
         """
+
+        # Define file paths
         self.input_file = input_file
         self.input_type = input_type
         self.output_bed = output_bed
         self.output_bigbed = output_bigbed
+        self.file_name = os.path.basename(input_file)
+        self.file_id = os.path.splitext(self.file_name)[0]
+        self.input_extension = os.path.splitext(self.file_name)[1]
+
         self.sample_name = sample_name
         self.genome = genome
-        cwd = os.getcwd()
-        if not rfg_config:
-            _LOGGER.info(f"Downloading refgenie files...")
-            self.rfg_config = os.path.join(cwd, "genome_config.yaml")
-        else:
-            self.rfg_config = rfg_config
-
         self.chrom_sizes = chrom_sizes
         self.check_qc = check_qc
-
+        self.rfg_config = rfg_config
+        # Define whether filter the input file to contain only the standard chromosomes
+        # If true, remove regions on ChrUn chromosomes
```

Comment:
I think this comments are unnecessary, and can be in cli, and can be added to docstrings